src/plugins/stepsform/stepsform.js

```diff
@@ -227,3 +227,80 @@ $document.on( "timerpoke.wb " + initEvent, selector, init );
 wb.add( selector );
 
 } )( jQuery, window, document, wb );
+
+( function( $, document ) {
+"use strict";
+
+// Fetch page language and set variables accordingly
+let relpreposition = " of ",
+	progressLabel = "Questionnaire progress:";
+
+// Define French progress label
+if ( wb.lang === "fr" ) {
+	relpreposition = " de ";
+	progressLabel = "Progression du questionnaire : ";
+}
+
+//Detect the enhancement of the quiz
+var quizSelector = ".provisional.wb-steps.quiz";
+
+//Initialize all instances
+var init = function( e ) {
```

Comment:
This whole function should be defined within the other init function instead of having its own init.